deploy/tools/envs-validator/test/.env.multichain

```
@@ -10,7 +10,7 @@ NEXT_PUBLIC_HOMEPAGE_STATS=['total_txs','wallet_addresses']
 NEXT_PUBLIC_API_DOCS_TABS=[]
 NEXT_PUBLIC_FEATURED_NETWORKS=http://example.com
 NEXT_PUBLIC_FOOTER_LINKS=http://example.com
-NEXT_PUBLIC_HOMEPAGE_HERO_BANNER_CONFIG={'background':['linear-gradient(90deg, rgb(232, 52, 53) 0%, rgb(139, 28, 232) 100%)'],'text_color':['rgb(255, 255, 255)']}
+NEXT_PUBLIC_HOMEPAGE_HERO_BANNER_CONFIG={'background':['linear-gradient(90deg, rgb(232, 52, 53) 0%, rgb(139, 28, 232) 100%)'],'text_color':['rgb(255, 255, 255)'],'text': 'Duck migration observer'}
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_HOMEPAGE_HERO_BANNER_CONFIG key should go before the NEXT_PUBLIC_HOMEPAGE_STATS key

(UnorderedKey)